cpp/tests/geometry/AxisAlignedBoundingBox.cpp

```diff
@@ -0,0 +1,299 @@
+// ----------------------------------------------------------------------------
+// -                        Open3D: www.open3d.org                            -
+// ----------------------------------------------------------------------------
+// The MIT License (MIT)
+//
+// Copyright (c) 2020 www.open3d.org
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+// IN THE SOFTWARE.
+// ----------------------------------------------------------------------------
+
+#include "open3d/geometry/BoundingVolume.h"
+#include "tests/UnitTest.h"
+
+using namespace open3d::geometry;
+using namespace ::testing;
+
+// Short names for fitting parametrized test cases into the 80 column limit
```

Comment:
We may remove this comment, replace `v_t` back to `Vector3d` (we may use `using Eigen::Vector3d`). 

For formatting:
You may run `make apply-style` from inside your build folder if you are using Mac or Ubuntu. [More about automated style formatting: [Here](http://www.open3d.org/docs/release/contribute/styleguide.html)].

Reply:
replaced `v_t` with `Vector3d`